components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -207,6 +207,13 @@ spec:
                 - prometheus
       securityContext:
         runAsUser: 0
+      // The PR builder image takes a little longer to start due to closing and building te binay from scratch.
+      // So to make sure both pods start at the same time we use the init container to check when the other pod is ready.
+      // Without this the scraping graphs will show slightly time shifted to each other.
+      initContainers:
```

Comment:
Should'nt this initContainer be a part of the 1st prometheus-deployment.
We want prometheus-test-{{ normalise .RELEASE }} to wait until PR has been built and started

Reply:
sorry I did'nt see the 3rd commit.
where the names are swapped

Reply:
yeah I tried initContainers, but didn't work well. Still big delay. the ready probe worked best

Reply:
this is the initial commit, but I removed it later.